docs/macos/readme.md

```diff
@@ -0,0 +1,169 @@
+# Check for prerequisites
+All requisites should be available for your distribution. The most important are:
+
+* [Git] (https://git-scm.com/downloads)
+* [Docker] (https://docs.docker.com/engine/installation/)
+* [Docker Compose] (https://docs.docker.com/compose/install/)
+
+# Installation
+- Clone and download the project to your working directory:
+
+```
+$ git clone https://github.com/npbtrac/enpii-dev-suite
+```
+
+- Go to the working directory:
+
+```
+$ cd enpii-dev-suite
+```
+
+The current working folder will be `enpii-dev-suite`. After all the prerequisites have been installed, run the following commands to start the installation of docker images.
+  - This command will check if there’s .env file, if there is not, it will init and generate an .env file. Then you can repair values that need to be updated located on your based folder.
+
+```
+$ sh scripts/init-env-file.sh
+```
+
+  - When environment variables have been updated, we will generate the `docker-compose.yml` and configuration files (Nginx conf - How to modify Nginx conf can be found [here](#setupcustomdomain)).
+
+```
+$ sh scripts/init-dev-suite.sh
+```
+
+  - Then download and bring up all the containers by the following command:
+
+```
+$ docker-compose up -d --build
+```
+
+- Wait for several mins and you'll have:
+  - Nginx (work as a webserver)
+  - PHP-FPM (PHP execution server)
+  - MySql
+  - Phpmyadmin
+- When it finishes downloading, we can check the list of all services by the command:
+
+```
+$ docker-compose ps
+```
+
+# Modify document root directory
+Firstly, create two files called `index.html` and `info.php` within folder `<path-to-document-root-directory>` and input the content:
+- index.html
+
+```
+<h1>Webserver works</h1>
+```
+
+- info.php
+
+```
+<?php echo phpinfo();
+```
+
+Later on, if your web page works, these contents will be shown.
+
+We have to modify the configuration for the `default.conf` file of Nginx container before launching. (View more details [here](#setupcustomdomain))
+
+# Set up custom domain
+- Before launching the web page, the configurations for the `default.conf` file of Nginx container need to be modified.
+  - Create a file called `default.conf` from sample `default.conf.example` with folder `etc/nginx` to repair document root. Each site will have its own Ngixn conf file.
+  - Later on, if you want to add configuration for other sites, simply create another conf file for each site and put it into folder `etc/nginx/sites-enabled`.
```

Comment:
There's a file `etc/nginx/sites-enabled/sample.conf.example`, make the guide to copy content from that file (you may need to `git pull origin master` to merge new update to your branch